src/libs/collect.c

```diff
@@ -86,8 +84,8 @@ typedef enum dt_lib_collect_cols_t
   DT_LIB_COLLECT_COL_ID,
   DT_LIB_COLLECT_COL_TOOLTIP,
   DT_LIB_COLLECT_COL_PATH,
-  DT_LIB_COLLECT_COL_COUNT,
   DT_LIB_COLLECT_COL_VISIBLE,
+  DT_LIB_COLLECT_COL_NOTHERE,
```

Comment:
It means that the file is missing from the current file system?
Not sure about the name, it took me some time to parse